pkg/workflow/analyticsWrapper.go

```diff
@@ -0,0 +1,99 @@
+package workflow
+
+import (
+	"fmt"
+	"net/http"
+	"strings"
+
+	"github.com/snyk/go-application-framework/pkg/analytics"
+)
+
+const analyticsPrefixSeparator = "::"
+
+func NewAnalyticsWrapper(a analytics.Analytics, prefix string) analytics.Analytics {
+	return &analyticsWrapper{next: a, prefix: fmt.Sprintf("%s%s", strings.ToLower(prefix), analyticsPrefixSeparator)}
```

Comment:
I don't like `next`. Where does that pattern come from? I've now seen it several times, and for me it's highly unexpressive.